apps/prairielearn/src/pages/instructorQuestions/instructorQuestions.html.ts

```diff
@@ -0,0 +1,41 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { QuestionsTable } from '../../components/QuestionsTable.html';
+
+export const QuestionsPage = ({ questions, resLocals }) => {
```

Comment:
Missing TypeScript types (this will go hand-in-hand with declaring types for `getQuestions`).

Reply:
done